packages/@okta/vuepress-site/code/angular/okta_angular_auth_js/index.md

```diff
@@ -22,28 +24,36 @@ If you do not already have a  **Developer Edition Account**, you can create one
 | -------------------  | --------------------------------------------------- |
 | Application Name     | OpenId Connect App *(must be unique)*               |
 | Login redirect URIs  | http://localhost:4200/callback                      |
-| Logout redirect URIs | http://localhost:4200/login                         |
+| Logout redirect URIs | http://localhost:4200                               |
 | Allowed grant types  | Authorization Code                                  |
 
 > **Note:** CORS is automatically enabled for the granted login redirect URIs.
 
 ## Create an Angular App
 
-To quickly create an Angular app, install the Angular CLI:
+To quickly create an Angular app, open a terminal and install the Angular CLI:
```

Comment:
If we are updating this like, lets take `quickly` out of it.   I dislike words like this, for people it is not `quick` or `easy` for